exercises/practice/forth/.meta/gen.go

```diff
@@ -12,84 +12,72 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	var j js
-	if err := gen.Gen("forth", &j, t); err != nil {
+	var j = map[string]interface{}{
+		"evaluate": &[]testCase{},
+		//TODO: add test with property `evaluateBoth` to forth_test.go and generate cases in cases_test.go
```

Comment:
Should this todo be part of this PR?

Reply:
Was not planned to be part of this PR. How to handle the following test case? `"scenarios": ["local-scope"],`:
```
 {
          "uuid": "3c8bfef3-edbb-49c1-9993-21d4030043cb",
          "description": "only defines locally",
          "comments": [
            "This test is meant to determine if instructions that define words are executed in the state of the program or locally.",
            "For example, in OOP-based approaches, the words should be defined in the same scope,",
            "or local to the instance the instructions were evaluated in."
          ],
          "scenarios": ["local-scope"],
          "property": "evaluateBoth",
          "input": {
            "instructionsFirst": [": + - ;", "1 1 +"],
            "instructionsSecond": ["1 1 +"]
          },
          "expected": [[0], [2]]
        }
```

Reply:
Not exactly sure what that test is supposed to be about. Maybe it would be good to ask in the Slack #discussions about this.
This is not blocking the merge though.